### PR TITLE
Hotifx/redirect and demographics chart height

### PR DESCRIPTION
**Pull request by nathanfranklin**
## Overview: ##

Hotfix to address some open issues:
* Update Readme
* Update redirect for onboarding
* Set max height for demographics chart (It was previously taking up too much vertical space)
* Fixed an issue in demographics chart instructions 
* Make Community Characteristics modal a little shorter

## Related Jira tickets: ##

None

## Testing Steps: ##
1. Check all pages
2. Confirm that community characteristics isn't too large.  It was too tall and we couldn't see the bottom when used in iframe
3. Confirm that when you select a county and view demographics that everything fits on the right panel (ie. demographics chart doesn't take up too much vertical space)

### CODE REVIEW

File: protx-client/src/components/ProTx/components/charts/MainPlot.module.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  font-weight: 700;
+  color: var(--color-blood-orange);
+}
+.loading-spinner {
+  margin: 5%;
+  margin-top: 40%;
+  text-align: center;
+  font-weight: 700;
+}
+.main-plot * {
```
Comment (nathanfranklin) [inline protx-client/src/components/ProTx/components/charts/MainPlot.module.scss:13]: I have this now in two spots CSS and in scss. so needs to be fixed before merging.

**Review by taoteg — APPROVED** [review]
Works perfectly for me. Nice work!

File: protx-client/src/components/ProTx/components/charts/MainPlot.module.scss (hunk 1 above)
Comment (nathanfranklin) [reply protx-client/src/components/ProTx/components/charts/MainPlot.module.scss:1]: fixed by @taoteg PR to this branch

**Review by sophia-massie — APPROVED** [review]
LGTM!

**Pull request merged**